nemo_skills/pipeline/convert.py

```diff
@@ -236,6 +236,7 @@ def convert(
     log_dir: str = typer.Option(None, help="Can specify a custom location for slurm logs."),
     exclusive: bool = typer.Option(False, help="If set will add exclusive flag to the slurm job."),
     check_mounted_paths: bool = typer.Option(False, help="Check if mounted paths are available on the remote machine"),
+    hf_home_check: bool = typer.Option(False, help="If False, skip checking HF_HOME in cluster_config."),
```

Comment:
we should do the check by default, so let's maybe call this `skip_hf_home_check` and if it's False, we check for env var?